content/en-US/api/roundRect.md

```diff
@@ -0,0 +1,113 @@
+---
+recipe: roundRect
+title: 'CanvasRenderingContext2D.roundRect()'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/roundRect
+specifications: https://html.spec.whatwg.org/#dom-context-2d-roundrect
+browser_compatibility: api.CanvasRenderingContext2D.roundRect
+---
+
+
+## Description
+
+The `roundRect` method of the `CanvasRenderingContext2D` interface adds a rounded rectangle to the 
+current sub-path.
+
+## Syntax
+
+`CanvasRenderingContext2D.roundRect(x, y, width, height, radii)`
+
+### Parameters
+<dl>
+  <dt><em>Parameter</em></dt>
+  <dd>Definition</dd>
+  <dt><em>x</em></dt>
+  <dd>A double that represents the x-axis (horizontal) coordinate of the rectangle's starting point.</dd>
+
+  <dt><em>y</em></dt>
+  <dd>A double that represents the y-axis (vertical) coordinate of the rectangle's starting point.</dd>
+
+  <dt><em>width</em></dt>
+  <dd>A double that represents the rectangle's width. Positive value means the path is drawn
+  clockwise; and negative value meeans it's drawn counterclockwise and flips the rectangle
+  horizontally along `y`, that means the radius values that applied to the left corners are now
+  applied to the right.</dd>
+
+  <dt><em>height</em></dt>
+  <dd>A double that represents the rectangle's height. Positive value means the path is drawn
+  clockwise; and negative value meeans it's drawn counterclockwise and flips the rectangle
+  vertically along `x`, that means the radius values that applied to the top corners are now applied
+  to the bottom.</dd>
+
+  <dt><em>radii</em></dt>
+  <dd>Each value `r` in `radii`  could be a number or an object with `{x, y}` properties. If `r` is 
```

Comment:
I feel like we should be consistent between "double" and "number". If we call it "double" above, why not call it "double" here as well?

Reply:
my fault, number is vague. i will remove all usage of number.